mip/cards.py

```diff
@@ -7,7 +7,8 @@
 """
 
 re_comment = re.compile('^\s{0,4}[cC](\s|$)')
-re_continuation = re.compile('^\s{5,}')
+re_continuation_spaces = re.compile('^\s{5,}')
+re_continuation_prev = re.compile('[^$]*&$')
```

Comment:
Formally, the ``&`` must be preceded by a space, as written in the manual, p. 3-4: "Alternatively, an &
(ampersand) preceded by at least one blank ending a line indicates data will continue on the
following card." Therefore the pattern can include the space before ``&``. I don't think that the ``&`` without leading space has another meaning, therefore the proposed pattern can be accepted.